openapi/components/schemas/QuoteChangeOrder.yaml

```
@@ -0,0 +1,486 @@
+type: object
+title: QuoteChangeOrder
+required:
+  - websiteId
+  - customerId
+  - items
+  - action
+  - orderId
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote for change order items.
+      On accept quote interim invoice is issued and linked to `invoiceId`.
```

suggestion: for readability
```suggestion
      When the accept quote action is used, an interim invoice is issued and linked to `invoiceId`.
```